sbi/inference/posteriors/likelihood_based_posterior.py

```diff
@@ -461,14 +485,15 @@ def posterior_potential(
         theta = ensure_theta_batched(torch.as_tensor(theta, dtype=torch.float32)).to(
             self.device
         )
+        theta, log_abs_det = self.transform(theta)
 
         log_likelihoods = LikelihoodBasedPosterior._log_likelihoods_over_trials(
             x=self.x,
             theta=theta,
             net=self.likelihood_nn,
             track_gradients=track_gradients,
         )
-        return log_likelihoods + self.prior.log_prob(theta)
+        return log_likelihoods + self.prior.log_prob(theta) + log_abs_det
```

Comment:
and for the prior, the prior log prob on theta is untransformed as well, no? not sure, but doesn't it need its own addition of log abs det then?

Reply:
Yes, the log-prob is untransformed. But one does not need two log-abs-dets. E.g. assume theta = sigmoid(theta')
```
p(theta') = 1 / J * p(theta) = 1 / J * (p(x|theta)p(theta))
```
J is the log-abs-det